app/models/cocktail.rb

```diff
@@ -2,15 +2,15 @@ class Cocktail < ApplicationRecord
   validates :name, :thumbnail, presence: true
 
   has_many :ratings, dependent: :destroy
-  def self.dashboard_five(user_id)
-#     left_outer_joins(:ratings)
-#       .select('cocktails.*, case when ratings.stars is null then 0 else ratings.stars end as rating')
-#       .where('ratings.user_id is null or ratings.user_id != ?', user_id)
-#       .order(Arel.sql('RANDOM()'))
-#       .limit(5)
-      left_outer_joins(:ratings)
-        .select('cocktails.*')
-        .order(Arel.sql('RANDOM()'))
-        .limit(5)
+  def self.dashboard_five(_user_id)
```

Comment:
small change? what does the `_` do?

Reply:
looks like this is a rubocop change. If a variable is not used it puts an underscore in front of it